src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/IconTintColor/IconTintColorBehavior.windows.cs

```diff
@@ -158,15 +158,13 @@ void ApplyImageTintColor(View element, WImage image, Color color)
 		var height = (float)image.ActualHeight;
 		var anchorPoint = new Vector2((float)element.AnchorX, (float)element.AnchorY);
 
-		// Hide possible visible pixels from original image.
-		// Workaround since the tinted image is added as a child to the current image and it's not possible to hide a parent without hiding its children using Visibility.Collapsed.
-		image.Width = image.Height = 0;
-
 		// Requested size requires additional offset to re-center tinted image.
-		var requiresAdditionalCenterOffset = element.WidthRequest != -1 || element.HeightRequest != -1;
-		var offset = requiresAdditionalCenterOffset ? new Vector3(width * anchorPoint.X, height * anchorPoint.Y, 0f) : Vector3.Zero;
+		var offset = new Vector3(width * anchorPoint.X, height * anchorPoint.Y, 0f);
 
 		ApplyTintCompositionEffect(image, color, width, height, offset, anchorPoint, uri);
+
+		// Hide possible visible pixels from original image by replaceing with a transparent image of the same size
+		image.Source = new WriteableBitmap((int)width, (int)height);
```

Comment:
if we cache this value, does it still work?

Reply:
Added caching of the blank image, and also made sure it was possible to return to the non-tinted image.

Reply:
I'm not sure why the build is failing, however. I've double checked and it builds and runs perfectly on my local machine. I also note that other PR builds are also failing with the same error messages.